src/main/java/core/mate/academy/service/MachineProducer.java

```diff
@@ -2,10 +2,11 @@
 
 import java.util.List;
 
-public interface MachineProducer {
+public interface MachineProducer<T> {
```

Comment:
lets add a bound, because you can create a MachineProducer<Cat>, lets prevent from it